mkt/langpacks/tests/test_models.py

```diff
@@ -44,6 +44,10 @@ def test_get_minifest_contents(self):
              'package_path': langpack.download_url,
              'developer': {'name': 'Mozilla'}})
 
+    def test_name(self):
+        langpack = LangPack(fxos_version='2.5.1')
+        eq_(langpack.name, u'English (US) language pack for Firefox OS 2.5.1')
```

Comment:
There is a comment above that also applies here: we should test this with translations, but need the weekly l10n extraction to be done and translations to be submitted first.